app/javascript/controllers/content_finder_controller.js

```diff
@@ -0,0 +1,107 @@
+import { Controller } from '@hotwired/stimulus'
+import * as jsonld from 'jsonld'
+import hljs from 'highlight.js/lib/core'
+import xml from 'highlight.js/lib/languages/xml'
+import json from 'highlight.js/lib/languages/json'
+
+export default class extends Controller {
+  static targets = ["content"]
+  static values = {
+    result: String,
```

Comment:
don't pass the content to display in the `result` value, just do `this.element.textContent` and pass the result in the HTML element content. 
Bellow an example 

```html
<div controller="content-finder">
#{@result}
</div>
```